src/components/Login/Login.tsx

```diff
@@ -19,7 +19,7 @@ interface LoginProps {
   toggleAuthForm: () => void;
 }
 export const Login: FC<LoginProps> = ({ toggleAuthForm }) => {
-  const { setLoginUser } = useContext(GlobalContext);
+  const setLoginUser = useGlobalStore(state => state.setUser);
```

Comment:
czasem dodajesz coś w nawiazie a czasem nie. zdecyduj jak wolisz i używaj jedną wersję. nie ma za bardzo znaczenia którą. po prostu żeby było spójne. 
Gdybyś miała jakoś dużo zagnieżdzeń w global state, wtedy warto dodać wyciągniecie tych zagnieżeń w nawiasie. a tutaj chyba lepiej nie. ale to możesz sama zdecydować